src/schema/imported/manifest.json

```diff
@@ -230,6 +230,10 @@
                       "preprocess": "localize"
                     }
                   }
+                },
+                "hidden": {
```

Comment:
I'm wondering if we need to "guard" this "hidden" property, I think that this isn't something that a non-system addon should actually be allowed to use.

Reply:
I can't find any documentation on it so I'm unsure tbh, I think this can be a follow-up as well.

Reply:
:+1: sure, that's totally fine, I mentioned it mainly to don't forget that I notice it and that we should investigate if the addon-linter should do something to ensure that it is handled correctly.